doc/overview.rst

```diff
@@ -41,8 +41,9 @@ Requirements
 Installing :mod:`scikit-sparse` requires:
 
```

Comment:
Can you quickly try older versions of everything and see what's actually supported?  I'd suggest supporting as many versions as possible without changing any code, but dropping any old version as soon as it becomes even mildly impractical.